examples/user_data/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13.1"
```

Comment:
do we really need to support so legacy version? a lot of issues or plan diffs are happening on legacy terraform versions (as in some point of time the split refresh phase from terraform)?

Reply:
@daroga0002 IIRC 0.15.4 and 1.0 have breaking behavior related to the way "outside changes" are tracked. This is one more reason to leave versions ranging so wide (some users can't simply upgrade so fast).